src/main/java/com/nerdapplabs/service/UserServiceImplement.java

```diff
@@ -116,4 +113,22 @@ public User mapRow(ResultSet rs, int rowNum) throws SQLException {
 		return listUsers;
 	}
 
+	@Override
+	public void sendEmail(User user) {
```

Comment:
by this, it will get registered email. either i have to intialize user inside method or i have to pass it in method